test/autoschedulers/adams2019/demo_generator.cpp

```diff
@@ -6,10 +6,10 @@ using namespace Halide;
 
 class ConvRelu : public Halide::Generator<ConvRelu> {
 public:
-    Input<Buffer<float, 4>> input{"input"};
```

Comment:
These changes look unintentional (same for the file below)

Reply:
Fixed here: https://github.com/halide/Halide/pull/7475